### ISSUE TITLE
"The program mps_genesim does not exist"

### ISSUE THREAD

**Issue by JBallGeo** [ergosimulation/mpslib#14]
Hi,

I am trying to run this program but I am getting the following error.

> ---------------------------------------------------------------------------
> Exception                                 Traceback (most recent call last)
> <ipython-input-10-32971e84e601> in <module>
>       1 # Run MPSlib
> ----> 2 O.run()
>       3 
>       4 
>       5 # Plot the results
> 
> /usr/local/lib/python3.8/site-packages/mpslib/mpslib.py in run(self, normal_msg, silent, thread)
>     493         if exe_path is None:
>     494             s = 'mpslib: The program {} does not exist or is not executable.'.format(exe_file)
> --> 495             raise Exception(s)
>     496             return -1
>     497         else:
> 
> Exception: mpslib: The program mps_genesim does not exist or is not executable.

**Comment by algocompretto** [comment]
I am having the same error while running the [Colab example](https://colab.research.google.com/github/ergosimulation/mpslib/blob/master/scikit-mps/examples/mpslib_in_google_colab.ipynb). I thought that maybe it is some environment and Path problem with Colab - but even when forcing the `mpslib.py` methods to search inside `content/mpslib` folder I get this error. Could you please share more on the error so we can troubleshoot our issue?

**Comment by ergosimulation** [comment]
Hi Gustavo.
I updated the pypi scikit-mps package as well as the colab notebook
example. pleas try again

https://colab.research.google.com/github/ergosimulation/mpslib/blob/master/scikit-mps/examples/mpslib_in_google_colab.ipynb

- Thomas
'

**Comment by algocompretto** [comment]
Yes, the Colab notebook is working as expected, thank you for your time!

**Comment by Miorat** [comment]
Hi, I have got the same issue. It woked fine with the previous version and then this happened after I updated to the latest mpslib-1.5. Coudl anyone help on how to fix this? 

Thanks. 

O.run()
################################################################
#
# mpslib: mps_genesim not found !!!!
# PLEASE ADD THE MPSLIB PROGRAM TO THE SYSTEM PATH
# OR ADD THE LOCATION OF THE MPSLIB PROGRAMS TO THE SYSTEM PATH
#
#################################################################
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
/tmp/ipykernel_5189/40755670.py in <module>
----> 1 O.run()

~/anaconda3/lib/python3.9/site-packages/mpslib/mpslib.py in run(self, normal_msg, silent, thread)
    583         if exe_path is None:
    584             s = 'mpslib: The program {} does not exist or is not executable.'.format(exe_file)
--> 585             raise Exception(s)
    586             return -1
    587         else:

Exception: mpslib: The program mps_genesim does not exist or is not executable.

**Comment by cultpenguin** [comment]
Do you still have this issue
Are you getting the error if you run your python code outside the mpslib source folder?
